src/Nethermind/Nethermind.Network.Test/PeerManagerTests.cs

```diff
@@ -133,15 +133,15 @@ public async Task Will_agree_on_which_session_to_disconnect_when_connecting_at_o
             if (firstDirection == ConnectionDirection.In)
             {
                 ctx.RlpxPeer.CreateIncoming(session1);
-                ctx.RlpxPeer.ConnectAsync(session1.Node);
+                _ = ctx.RlpxPeer.ConnectAsync(session1.Node);
```

Comment:
This silences C# compiler warning about un-awaited async call.  I'm guessing the lack of await was intentional, the `_ =` just makes it obvious/explicit.

Reply:
I see, possible as it tries to fire and forget connections for testing